package/gluon-mesh-layer3-common/luasrc/usr/lib/lua/gluon/l3.lua

```diff
@@ -7,7 +7,9 @@ local M = {}
 function M.node_client_prefix6()
 	local key = "gluon-l3roamd.node_client_prefix6"
 	local prefix_seed = util.domain_seed_bytes(key, 7)
-	return ("fd" .. prefix_seed):gsub(("(%x%x%x%x)"):rep(4), "%1:%2:%3:%4" .. "::/64")
+	local prefix = ("fd" .. prefix_seed):gsub(("(%x%x%x%x)"):rep(4), "%1:%2:%3:%4" .. "::/64")
+
+	return prefix
```

Comment:
Parentheses are sufficient to drop the second return value:
```suggestion
	return (("fd" .. prefix_seed):gsub(("(%x%x%x%x)"):rep(4), "%1:%2:%3:%4" .. "::/64"))
```